exercises/ex1.cpp

```diff
@@ -11,38 +40,7 @@ int main()
   cout << "Enter week number(1-7): " << endl;
   cin >> week;
 
-  if (week == 1)
-  {
-    cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
-    cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
-    cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
-    cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
-    cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
-    cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
-    cout << "Sunday" << endl;
-  }
-  else
-  {
-    cout << "Invalid input! Please enter week number between 1-7." << endl;
-  }
+  printDay(week);
 
   return 0;
```

Comment:
`return 0` non è molto significativo e non vi è una determinata semantica. Per migliorare la leggibilità potresti definire:

```cpp
#define EXIT_SUCCESS 0;
```
E utilizzarla nel seguente modo.

```suggestion
  return EXIT_SUCCESS;
```